sdk/cognitivelanguage/ai-language-text/review/ai-language-text.api.md

```diff
@@ -878,6 +1067,7 @@ export interface TextAnalysisSuccessResult {
 
 // @public
 export interface TextDocumentBatchStatistics {
+    [property: string]: any;
```

Comment:
what is this property on statistics?

Reply:
This means the service could return extra things that we don't know their names yet, I will follow-up on this with the service team once we get the deployment done.

Reply:
Ah I see: https://github.com/Azure/azure-rest-api-specs/blob/main/dev/cognitiveservices/data-plane/Language/common.json#L375

Yeah I wonder if that was intentional